src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].name();
```

Comment:
Calling `Colors.values()` inside a loop or a frequently called method can be inefficient because it creates a new array every time it is called. Consider storing the result in a constant.